README.md

```diff
@@ -79,6 +79,6 @@ The project follows the same release process as the other libraries in the MetaM
 - For the global `wallet` type to work, you have to add the following to your `tsconfig.json`:
   ```json
   {
-    "files": ["./node_modules/@metamask/snap-types/global.d.ts"]
+    "files": ["./node_modules/@metamask/snaps-types/global.d.ts"]
   }
   ```
```

Comment:
This is no longer necessary. The global types are automatically imported as long as the snap is using an import from `snaps-types`.

Reply:
Is that true for `0.27.1`?

Reply:
Yes. It's part of [this commit](https://github.com/MetaMask/snaps-monorepo/commit/b3ce1ffdf7bd5ed125d53f3fd489d3adfacfb9fb), which was first released in 0.27.0.

Is there any reason for us not to use the latest version, though?

Reply:
Ah sweet, I didn't realize that. I don't want to use the latest version as there may be breaking changes not part of the `10.25.0` release.